database/seeds/CourseSeeder.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class CourseSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('courses')->insert([
+            [
+                'title' => 'Khoá học 1',
+                'image' => 'course1.png',
+                'description' => 'Chúng ta đăng ký tài khoản với hệ thống thì trường password thì phải
+                    tối thiểu 60 ký tự và tối đa 255 ký tự nhé. Chúng ta cũng có thể đổi thay đổi cho
+                    trường remember_token là nullable() khi migrate bảng users.Ak mình nới thêm chút về
+                    cái trường remember_token - đây là trường sẽ lưu token để giành cho chức năng
+                    Remember me mình sẽ giới thiệu ở phần sau nhé. Đừng lo lắng nhé nếu
+                    mà chúng ta cứ sử dụng như default thì chúng ta chẳng cần động vào file
+                    config/auth.php này làm gì !!!!',
```

Comment:
4 arrays khác gì nhau đâu? sao ko cho vào `for` ?
dùng `faker`

Reply:
mình sẽ sửa ngay ạ